### ISSUE TITLE
Images does not match for the same hash but different sizes

### ISSUE THREAD

**Issue by e1himself** [e1ven/Robohash#9]
Hello!

For some reason robohash generates different image for a hash when specifying different sizes.

I've tried different sizes and found, that images does not match for 64x and 63x sizes. 

Yellow robot:
* https://robohash.org/fd7a4c9e24ae4ed592abba0098c18859395aba2d?size=63x63 
* https://robohash.org/fd7a4c9e24ae4ed592abba0098c18859395aba2d?size=64x64

Purple robot:
* https://robohash.org/fd7a4c9e24ae4ed592abba0098c18859395aba2d?size=16x16
* https://robohash.org/fd7a4c9e24ae4ed592abba0098c18859395aba2d?size=65x65
* https://robohash.org/fd7a4c9e24ae4ed592abba0098c18859395aba2d?size=128x128
* ... and all other sizes I've tried.

Thank you.

![2015-04-02 10 29 18](https://cloud.githubusercontent.com/assets/370680/6959636/cc552378-d923-11e4-806b-de0817b25439.png)

**Comment by e1ven** [comment]
Sorry for the trouble!
It should be fixed for you now, but the solution isn't a clean one.
I believe it happened a bit ago, when I needed to add some additional servers to handle capacity.

It looks like the code that's sorting still isn't relying on filename order - It's using inodes someplace.
When I run the code on new systems, I get a different robot :/

For now, I've removed the offending systems. 
I'll try to go back and fix the code sometime soon, but I'm offended looking at it - I did such a bad job of writing this ;)  I really ought to rewrite the whole thing.

But in any event, this should fix the problem for you now, until I can fix it right.
